### PR TITLE
Add Scaleway to the list of provider

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

**Comment by remyleone** [comment]
@savitharaghunathan fixed

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ The following production environment solutions table lists the providers and the
 | [Pivotal](https://pivotal.io/) | | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service) | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service)           |      |          |
 | [Platform9](https://platform9.com/)         | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)             |       | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)            |  &#x2714;    |  &#x2714;    |   &#x2714;
 | [Rancher](https://rancher.com/)         |              |   [Rancher 2.x](https://rancher.com/docs/rancher/v2.x/en/)     |             |  [Rancher Kubernetes Engine (RKE)](https://rancher.com/docs/rke/latest/en/)    |          | [k3s](https://k3s.io/)
+| [Scaleway](https://www.scaleway.com/)        | [Scaleway Kapsule (Beta)](https://www.scaleway.com/en/kubernetes/)    | [Scaleway Instances](https://www.scaleway.com/en/virtual-instances/) |      |     |            |
```
Comment (sftim) [inline content/en/docs/setup/_index.md]: Would it be OK to hold off until the Kapsule service has left public beta?
(My opinion): readers looking for a turnkey solution would want a list of stable services.

**Comment by gochist** [comment]
@remyleone Hello! Could you please limit pull requests to one language per PR following the [guide](https://kubernetes.io/docs/contribute/start/#submit-a-pull-request)? 
/assign @zparnold

**Comment by zparnold** [comment]
/close 
per @gochist 's comment @remyleone please re-open in each language. However, I also agree with @sftim that turnkey solutions should probably themselves be stable (and not beta)

**Pull request closed without merging**